docs/containerd-2.0.md

```diff
@@ -0,0 +1,105 @@
+
+This release adds the `Update` API definition for the sandbox controller. (`/containerd.services.sandbox.v1.Controller/Update`)
+
+`Update` defines the API that sandbox plugins will implement to allow for modifications to an existing sandbox's attributes such as [sandbox] specification, runtime, extensions, and labels.
+
+### Adds OTEL environment variable configuration support for containerd's built-in tracing plugin
+
+This release adds support for the standard environment variables defined in the OpenTelemetry [specification](https://opentelemetry.io/docs/specs/otel/configuration/sdk-environment-variables/) to the built-in containerd tracing plugin.
+
+Implementation note: by default OTEL will try to connect to `https://localhost:4318` if no endpoint is set. In contrast, the containerd tracing OLTP plugin will be disabled if no endpoint is configured.
+
+i.e. both `OTEL_SDK_DISABLED` and one of either `OTEL_EXPORTER_OTLP_ENDPOINT` or `OTEL_EXPORTER_OTLP_TRACES_ENDPOINT` must be set in the containerd environment for the tracing plugin to be enabled.
+
+### Enables NRI by default
+
+This release enables NRI by default. To disallow NRI functionality, users will need to explicitly disable the common NRI plugin in containerd configuration. See [Disabling NRI Support in Containerd](../docs/NRI.md#disabling-nri-support-in-containerd) for details.
+
+### Adds Intel ISA-L's igzip support
+
+This release adds support for Intel ISA-L's igzip to the containerd client. If found, the containerd client will use igzip for gzip decompression. Similar to other external gzip implementations (e.g. pigz), users can disable igzip using the `CONTAINERD_DISABLE_IGZIP` variable in the containerd client process environment.
+
+## What's breaking
+
+### Disables support for Docker Schema 1 images
+
+Pulling Docker Schema 1 images has been disabled by default in containerd v2.0. Users can re-enable previous behavior by setting `CONTAINERD_ENABLE_DEPRECATED_PULL_SCHEMA_1_IMAGE=1`; however, users are **recommended** to begin migration to Docker Schema 2 or OCI images. Support for Docker Schema 1 images will be fully removed in containerd v2.1.
+
+### Disallows `io_uring_*` syscalls by default
+
+The following syscalls (`io_uring_enter`, `io_uring_register`, and `io_uring_setup`) have been removed from the default Seccomp profile allowlist.
+
+### Moves containerd client to subpackage
+
+The containerd client has moved to the `github.com/containerd/containerd/v2/client` sub-package. See [getting-started.md](../docs/getting-started.md#implementing-your-own-containerd-client) for a working example for building on top of the containerd client.
+
+### Removes `LimitNOFILE` from `containerd.service`
+
+The reference `containerd.service` will rely on the systemd default `LimitNOFILE` configuration in place of the previously set limit (`infinity`). On platforms with systemd prior to v240, packagers should explicitly configure `LimitNOFILE=1024:524288`.
```

Comment:
This needs a bit more explanation (and maybe a link to the issue that has a long discussion here).  containerd's rlimits are inherited by containers, so the daemon's `LimitNOFILE` affects `RLIMIT_NOFILE` within containers.

Reply:
Thanks for the help with the wording here. Expanded this to highlight the risk and impact to containers.